arch/arm/core/swap_helper.S

```diff
@@ -427,7 +427,7 @@ SECTION_FUNC(TEXT, z_arm_svc)
 
     ldr r1, [r0, #24]   /* grab address of PC from stack frame */
     /* SVC is a two-byte instruction, point to it and read encoding */
```

Comment:
change this comment to reference reading the lower byte of SVC

Reply:
Right, thanks. Fixed now